### PR TITLE
Image Filenames and Accessibility

### LINKED ISSUE #225: Image Filenames and Accessibility
Images should be renamed from `imageXX` to a meaningful name.
Alt text should also be added in the RST.

### PR DESCRIPTION
**Pull request by jasondaming**
Closes #225

### CODE REVIEW

**Comment by jasondaming** [comment]
@prateekma & @Daltz333 hopefully this is easier to review

**Review by Daltz333 — DISMISSED** [review]
This looks fantastic, thank you a ton. This is great work. I'd like at least two others to give it a review before merging since it's so widely touching.

**Review by Modelmat — CHANGES_REQUESTED** [review]
I did the hardware section and then gave up... There's *a lot* there.

Most of the alt-texts seem to have similar issues in that it's not a description of the actual content of the image, but a description of what the image is, which means the alt text is not-so-useful.

Also, do we have a way to create a check for the existence of alt-text on images?

**Comment by jasondaming** [comment]
In general I got feedback that using "shows/showing" was superfluous and I agree.  Some things that are a picture of an object are hard to describe unless I actually try to describe the physical properites of the object, but is that helpful?

As you said it was a ton of work.  Some things may have just been due to my brain melting towards the end!

File: source/docs/hardware/hardware-basics/status-lights-ref.rst
Comment (jasondaming) [reply source/docs/hardware/hardware-basics/status-lights-ref.rst:135]: What benefit does "showing" add here?  I originally had this terminology but cut it due to comments.  I could add status

File: source/docs/hardware/hardware-basics/wiring-pneumatics.rst
Comment (jasondaming) [reply source/docs/hardware/hardware-basics/wiring-pneumatics.rst:42]: Why does this image need a width?  @Daltz333 is very anti width.  I like it because (for some reason) it makes the image clickable.

File: source/docs/hardware/sensors/accelerometers-hardware.rst
Comment (jasondaming) [reply source/docs/hardware/sensors/accelerometers-hardware.rst:30]: I think this doesn't really clarify anything

File: source/docs/hardware/hardware-basics/wiring-pneumatics.rst
Comment (jasondaming) [reply source/docs/hardware/hardware-basics/wiring-pneumatics.rst:42]: added of the PCM

File: source/docs/hardware/hardware-basics/status-lights-ref.rst
Comment (jasondaming) [reply source/docs/hardware/hardware-basics/status-lights-ref.rst:135]: Modified to include the extra details

File: source/docs/hardware/sensors/accelerometers-hardware.rst
Comment (jasondaming) [reply source/docs/hardware/sensors/accelerometers-hardware.rst:40]: added printed.

File: source/docs/hardware/sensors/analog-inputs-hardware.rst
Comment (jasondaming) [reply source/docs/hardware/sensors/analog-inputs-hardware.rst:22]: used

File: source/docs/hardware/sensors/analog-inputs-hardware.rst
Comment (jasondaming) [reply source/docs/hardware/sensors/analog-inputs-hardware.rst:42]: done

File: source/docs/hardware/sensors/analog-potentiometers-hardware.rst
Comment (jasondaming) [reply source/docs/hardware/sensors/analog-potentiometers-hardware.rst:18]: done

File: source/docs/hardware/sensors/digital-inputs-hardware.rst
Comment (jasondaming) [reply source/docs/hardware/sensors/digital-inputs-hardware.rst:34]: changed to normally open.  It is discussed in other articles but not this one.

File: source/docs/hardware/hardware-basics/status-lights-ref.rst
Comment (Modelmat) [reply source/docs/hardware/hardware-basics/status-lights-ref.rst]: I think that "showing" is generally superfluous - however, I'm not sure what WPILib is going for with alt-text.

The way I understood alt-text for accessibility (i.e. you are blind and need a screen-reader) is that the alt-text should describe the contents of the image (i.e. it describes what the actual picture is of and the important details within in).

Hence, "Highlights the position of the 12V and 5V status lights." isn't really useful as it requires the knowledge that you have a VRM in the picture.

**Review by Daltz333 — APPROVED** [review]

**Review by Daltz333 — CHANGES_REQUESTED** [review]
Can you force-push so a new RTD build is triggered?

**Review by sciencewhiz — APPROVED** [review]

**Review by Daltz333 — APPROVED** [review]
I'm approving, and will merge in the next day or so assuming no other concerns are brought up.

**Pull request merged**
